setup-scripts/health-checker.py

```diff
@@ -210,5 +211,60 @@ async def check_health():
             client, message, file=failed_records_file, force_notify=force_notify
         )
 
+# check_alerts checks the alerts returned from siad's daemon/alerts API
+async def check_alerts():
+    print("\nChecking portal siad alerts...")
+
+    try:
+        alerts_res = requests.get("http://localhost:9980/daemon/alerts",headers={"User-Agent":"Sia-Agent"}, verify=False)
+        alerts_json = alerts_res.json()
+    except:
+        trace = traceback.format_exc()
+        print("[DEBUG] check_alerts() failed.")
+        return await send_msg(
+            client, "Failed to run the checks!", file=trace, force_notify=True
+        )
+
+    alerts =  alerts_json['alerts']
+    critical_alerts =  alerts_json['criticalalerts']
+    error_alerts =  alerts_json['erroralerts']
+    warning_alerts =  alerts_json['warningalerts']
+    siafile_alerts = []
+    siafile_alert_message = "The SiaFile mentioned in the 'Cause' is below 75% redundancy"
+
+    # Check for siafile alerts in alerts. This is so that the alert severity
+    # can change and this doesn't need to be updated
+    for alert in alerts:
+        if alert['msg'] == siafile_alert_message:
+            siafile_alerts.append(alert)
+
+
+    ################################################################################
+    ################ create a message
+    ################################################################################
+
+    message = ""
+    force_notify = False
+
+    if len(critical_alerts) > 0:
+        message += "{} CRITICAL Alerts found! ".format(len(critical_alerts))
+        force_notify = True
+    if len(error_alerts) > 0:
+        message += "{} Error Alerts found! ".format(len(error_alerts))
+        force_notify = True
```

Comment:
If we add these there will be a lot of duplication between these alerts and what's reported based on the logs.
I'm not sure what's the best approach. Would everything interesting be reported by these alerts? If so, we can use that and stop monitoring the log file. OTOH, the log file is available even when `siad` is struggling for some reason (but not when it's completely dead because then the container is down).

Or maybe we can scan these alerts only for specific problems we want to monitor and rely on all other issues being logged as well?

Reply:
`log.Critical` is different that Critical alerts. 

`log.Critical` is used when something terrible has happened and siad shouldn't continue.

Critical Alerts are alerts that the user should be aware of and they need to take specific action on to resolve as siad can't resolve them on their own.   

Same idea with alert errors, they shouldn't be overlap with the logs check. The logs check is scanning stdout right? not the actual `.log` files of the siad instance correct? 

Also I quickly checked, there is only 1 Error alert and 6 Critical Alerts. 5 of the critical alerts are for hosts so our portals would not see those ever.

So then the Error and Critical alerts that this check would be picking up are related to contract formation and renewal which currently don't have `log.Criticals`. They do log to the `contractor.log` file with `log.Debugln("SEVERE")` but since it is not `log.Severe` the log check wouldn't pick it up. 

All to say, this isn't duplicating anything that the log check should be notifying us about unless I'm mistaken in how the log check is functioning.

Reply:
> Same idea with alert errors, they shouldn't be overlap with the logs check. The logs check is scanning stdout right? not the actual .log files of the siad instance correct?

Yes, that's correct. Technically, I believe it scans both stdout and errout but it's all the same in this context.

Alright, great, then we're getting more visibility without overlap!